user_test.go

```diff
@@ -136,3 +136,14 @@ func TestListUserDevices(t *testing.T) {
 }
 
 // NOTE RevokeUserDevice is not tested because it is impossible to spoof webauthn to create a device to then revoke
+
+func TestSignOutUser(t *testing.T) {
+	psg, err := passage.New(PassageAppID, &passage.Config{
+		APIKey: PassageApiKey, // An API_KEY environment variable is required for testing.
+	})
+	require.Nil(t, err)
+
+	result, err := psg.SignOut(CreatedUser.ID)
```

Comment:
To fix the failing test, I think this would either need to be 
```suggestion
	result, err := psg.SignOut(PassageUserID)
```
or this can be moved before the TestDeleteUser function.